### PR TITLE
Minimal solution to raising of warning instead of writing them

### PR DESCRIPTION
**Pull request by asanin-epfl**
For presentation purpose only. For #262

### CODE REVIEW

**Comment by tomdele** [comment]
If we choose this path (that is, allowing to raise if warnings) I am fine with the implementation. 
Just the names should change.

**Comment by asanin-epfl** [comment]
@mgeplf , @tomdele this one is ready for review. I can't edit coverage plan. It fails due to 0.01% decrease.

File: include/morphio/errorMessages.h
Comment (tomdele) [inline include/morphio/errorMessages.h:16]: for homogeneity I think you should name it : 
`set_raise_warnings`

Comment (asanin-epfl) [reply include/morphio/errorMessages.h]: indeed, thank you!

**Comment by asanin-epfl** [comment]
I will be waiting until #263 is merged, so we will have better tests here.

**Comment by tomdele** [comment]
I think you can approve the #263 we should be good

**Comment by asanin-epfl** [comment]
too much responsibility for me, I rather wait for Mike's final response.

**Review by tomdele — APPROVED** [review]

**Comment by mgeplf** [comment]
Looks great, thanks for moving forward on this.

**Pull request merged**
